### PR TITLE
Make it pip installable

### PR DESCRIPTION
**Pull request by ttedeschi**
This PR aims at making this package pip installable

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+setup(
+    name='tdrstyle',
+    version='0.1.0',    
+    description='CMS plotting style ROOT macros',
+    url='https://github.com/cms-cat/cms-root-style',
+    author='CMS Common Analysis Tools',
+    #author_email='',
+    license='BSD 2-clause',
+    packages=['tdrstyle'],
```
Comment (anmalara) [inline setup.py:11]: Do we want to rename this into CMSstyle? I think tdrstyle is quite an old naming.
Thoughts?

Comment (ttedeschi) [reply setup.py:11]: Yes I agree @andrzejnovak @lenzip @clelange what do you think?

Comment (lenzip) [reply setup.py:11]: CMSstyle seems appropriate.
Is BSD the most appropriate license?

Comment (ttedeschi) [reply setup.py:11]: > CMSstyle seems appropriate. Is BSD the most appropriate license?

@lenzip At the moment that is just a place-holder.. the whole repo at the moment is under MIT.. what do you think would be the most appropriate?

Comment (lenzip) [reply setup.py:11]: I think MIT is ok

File: CMSstyle/CMSstyle.py
[the remark this answers is not in the document]
Comment (lenzip) [reply CMSstyle/CMSstyle.py:1]: I would have a slight preference for renaming, but let's hear opinions.

[the remark this answers is not in the document]
Comment (anmalara) [reply CMSstyle/CMSstyle.py:1]: makes sense to me too :-) Thanks

**Review by ttedeschi — COMMENTED** [review]
Renaming done here

**Pull request merged**